recipes/yodapy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.2.3b0" %}
+
+package:
+  name: yodapy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/y/yodapy/yodapy-{{ version }}.tar.gz
+  sha256: 48052e96bdfa1e065d18a3bd91c73100d789e35b8f37c585e7914e1403b5e1ab
+
+build:
+  number: 0
+  noarch: python
```

Comment:
I checked upstream and it seems that we can make this `noarch`, let me know if I'm missing something.